hooks/terraform-validate.sh

```diff
@@ -9,7 +9,7 @@ export PATH=$PATH:/usr/local/bin
 
 for dir in $(echo "$@" | xargs -n1 dirname | sort -u | uniq); do
   pushd "$dir" >/dev/null
-  terraform init -backend=false "$dir"
-  terraform validate "$dir"
+  terraform init -backend=false
+  terraform validate
```

Comment:
Is `dir` not necessary because of the `pushd` above?

Reply:
Yup. In fact, `dir` is a relative path, which is why it is breaking.